JSDOMParser.js

```diff
@@ -1,4 +1,3 @@
-/*eslint-env es6:false*/
```

Comment:
As far as I can tell this wasn't doing anything meaningful - there is already ES6 use in `Readability.js` (notably `Set` and `let`) so that ship has sailed.